core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -200,8 +200,9 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       this.heading = 'Review Translation Contributions';
     }
     delete this.suggestionIdToContribution[this.initialSuggestionId];
-    this.remainingContributionIds = Object.keys(
-      this.suggestionIdToContribution);
+    this.remainingContributionIds = Object.keys(this.suggestionIdToContribution)
```

Comment:
We have `this.suggestionIdToContribution` and the suggestion ID here, so I think we should just compute `remainingContributionIds` and `skippedContributionIds` directly here instead of in the other component.

Reply:
Done.

Reply:
Can you also add a test for this so that we don't run into this in the future?

Reply:
Done.